TYPO3.Flow/Classes/TYPO3/Flow/Http/RequestHandler.php

```diff
@@ -111,8 +116,8 @@ public function handleRequest()
             $this->request->setBaseUri(new Uri($this->settings['http']['baseUri']));
         }
 
-        $componentContext = new ComponentContext($this->request, $this->response);
-        $this->baseComponentChain->handle($componentContext);
+        $this->componentContext = new ComponentContext($this->request, $this->response);
```

Comment:
But that again is also not easily possible, it would move the ComponentContext and ALL components (also user land) to a pre boot time, which has tremendous implications like no Configuration, no Persistence, no Caches no nothing basically...